source/public_collections_advanced_search.md

```diff
@@ -0,0 +1,97 @@
+# Public Collections Advanced Search
+
+**_Note that the user-friendliness of advanced searching will improve in the
+coming months._**
```

Comment:
Let's not give this time frame as I'm not sure we'll be able to stick to it. Perhaps
> Please note that the search capability is a work in progress, and we plan to add a more user-friendly syntax in future releases.